ostelco-ios-client/ViewControllers/Home/SettingsViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  SettingsViewController.swift
+//  ostelco-ios-client
+//
+//  Created by Prasanth Ullattil on 23/05/2019.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    @IBOutlet private weak var versionLabel: UILabel!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        var appVersion = ""
+        if let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            appVersion = "App Version \(version)"
+        }
+        if let buildNumber = Bundle.main.infoDictionary?["CFBundleVersion"] as? String {
```

Comment:
There's a `kCFBundleVersionKey` constant so you don't have to hard-code this string (stupidly, there's not one for the short version)